src/main/java/com/meetup/server/place/domain/value/GoogleReview.java

```diff
@@ -24,9 +25,14 @@ public static GoogleReview from(GoogleSearchTextResponse.Review review) {
         return GoogleReview.builder()
                 .rating(review.rating())
                 .content(review.originalText().text())
+                .translatedContent(null)
                 .author(review.authorAttribution().displayName())
                 .authorProfileImage(review.authorAttribution().photoUri())
                 .publishTime(publishTime)
                 .build();
     }
+
+    public GoogleReview updateContent(String translated) {
```

Comment:
`updateContent` 는 데이터를 수정하는 뉘앙스에 가까운데, 저희는 번역한 내용을 넣어주기 때문에 메소드명을 수정해봐도 좋을 거 같아요!

찾아보니 immutable한 객체의 setter를 `with` 를 붙여 만든다고 해서 `withTranslatedContent`도 괜찮을 거 같아요!

```suggestion
    public GoogleReview withTranslatedContent(String translated) {
```

참고) [java.time package docs](https://docs.oracle.com/javase/8/docs/api/java/time/package-summary.html), [stack overflow](https://stackoverflow.com/questions/9570015/java-pseudo-setter-method-for-immutable-classes)

<img width="1074" alt="스크린샷 2025-06-27 오후 4 05 58" src="https://github.com/user-attachments/assets/c114f9c4-2fb6-43f0-9d03-d1299c352151" />